runtest-moduleapi

```diff
@@ -35,4 +35,5 @@ $TCLSH tests/test_helper.tcl \
 --single unit/moduleapi/hash \
 --single unit/moduleapi/zset \
 --single unit/moduleapi/stream \
+--single unit/pubsub \
```

Comment:
i don't think we want to add the non-module test unit to the module api test list,
and i don't think we want to add any reference to modules in the non-modules tests (i.e. pubsub.tcl).

these are currently two separate test runs. as you saw, the normal unit test doesn't even bother to build the modules, and that's on purpose.
let's copy whatever parts you want from unit/pubsub.tcl into moduleapi/keyspace_events.tcl